### PR TITLE
:sparkles: component / taxe

### PR DESCRIPTION
**Pull request by guillaumecrespel**
issue #30 

- [x] basic style

- [x] logic component

- [x] wait #24 to code business

### CODE REVIEW

**Comment by guillaumecrespel** [comment]
image

File: ui/ui-expenses/src/screens/prices/prices.jsx [hunk 1]
```diff
@@ -1,21 +1,31 @@
...
 import PropTypes from 'prop-types'
 import Price from './price'
+import Taxe from './taxe'
 import styles from './prices.styles'
 
-const Prices = ({ prices }) => {
-  const defaultSelected = prices[0]
-  return (
-    <div className={styles.prices}>
+const taxes = [20, 10, 5.5]
```
Comment (guillaumecrespel) [inline ui/ui-expenses/src/screens/prices/prices.jsx]: hard coded atm

File: ui/ui-expenses/src/screens/prices/taxe/taxe.styles.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  },
+  input: merge(
+    {
+      position: 'absolute',
+      opacity: 0,
+    },
+    select('& + div', {
+      padding: '.7em',
+      minWidth: '4em',
+      backgroundColor: colors.green,
```
Comment (guillaumecrespel) [inline ui/ui-expenses/src/screens/prices/taxe/taxe.styles.js]: this color it's temporary, don't worry 😄

**Comment by guillaumecrespel** [comment]
on second time, when we thinking about design, this component will be merged with price of course

**Comment by fabienjuif** [comment]
### TODO
 - [ ] add `0%`
 - [ ] add `n/a`

File: ui/ui-expenses/src/screens/prices/prices.reactions.js [hunk 3]
```diff
@@ -1,10 +1,13 @@
 export const add = ((action, store, { http }) => {
+  const taxe = Number.parseFloat(store.ui.taxe.get(), 10)
```
Comment (fabienjuif) [inline ui/ui-expenses/src/screens/prices/prices.reactions.js]: format || ''

**Pull request merged**